pkgs/top-level/all-packages.nix

```diff
@@ -624,6 +624,8 @@ with pkgs;
 
   domine = callPackage ../tools/misc/domine { };
 
+  dpp = callPackage ../development/libraries/dpp { };
```

Comment:
Please use [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md).

Reply:
I tried doing this while following the structure from the readme
    dpp = callPackage ../by-name/dp/dpp/package.nix { };
  
  
    "pkgs/by-name/dp/dpp/package.nix" being the path where the modified derivation is
  
   but I did something wrong, can you help me identify my error?

Reply:
Completely remove the line from `all-packages.nix`.